README.md

```diff
@@ -1,6 +1,6 @@
 # Simple S3 Resource for [Concourse CI](http://concourse.ci)
 
-Resource to upload files to S3. Unlike the [the official S3 Resource](https://github.com/concourse/s3-resource), this Resource doesn't care about files being versioned.
```

Comment:
Oh... yeah, I knew you were talking about file versioning, I think I sent a slightly errant link in the last comment.   The official resource also looked like it had quite a bit of recent development.  Perhaps when you started this the official resource didn't cut it. :)